### DISCUSSION TITLE
Is it possible to map the elastic search volume?

### DISCUSSION THREAD

**Discussion by patrick-5546** [nextcloud/all-in-one#1425] [Questions] [answered]
The elastic search volume (`/var/lib/docker/volumes/nextcloud_aio_elasticsearch`) quickly grows larger than the rest of my images and volumes combined. Is there a way that I can map this volume to a folder so that it doesn't fill my Docker vdisk `
/mnt/user/system/docker/docker/docker.img`? I am running Unraid.

**Comment by szaimen** [comment, marked as answer]
No that is not possible afaik. However you could try to create the elasticsearch volume before AIO is started the first time by doing like below but I think AIO will run into an issue with that when starting the containers (feel free to do so nonetheless and post the error log here that you get so that we can adjust it in the next release.):

```
docker volume create \
--driver local \
--name nextcloud_aio_elasticsearch \
-o device="/your/data/path" \
-o type="none" \
-o o="bind"
```
(of course dont forget to adjust `/your/data/path`.

**Reply by patrick-5546** [reply]
By the first time do you mean from a fresh install? Is there any way I could try this out with my existing install without any risk of corrupting it?

**Reply by szaimen** [reply]
I fear this will get much more complicated as you would need to first copy the data over and then do as suggested afterwards...

**Reply by patrick-5546** [reply]
What if I don't care about the data (I'm assuming you are talking about the elasticsearch volume)? If I delete the existing volume and create a new volume, would it not restart indexing?

**Reply by szaimen** [reply]
I would really not recommend trying to reset the fts database. It involves multiple things and last time I tried, I failed.

So moving the files over is you best bet, imho.

**Reply by szaimen** [reply]
I've succeeded in resetting the index. See https://github.com/nextcloud/all-in-one/discussions/2509
